src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -336,5 +347,16 @@ func (updated *Config) updateFrom(updateBytes []byte) error {
 	for _, server := range servers {
 		updated.Client.FrontedServers = append(updated.Client.FrontedServers, server)
 	}
+
+	// Same with global whitelist
+	// Would be nice to combine this with the above at some point!
+	wlDomains := make(map[string]bool)
+	for _, domain := range updated.Client.Whitelist.Cloud {
+		wlDomains[domain] = true
+	}
+	updated.Client.Whitelist.Cloud = make([]string, 0, len(wlDomains))
+	for domain, _ := range wlDomains {
+		updated.Client.Whitelist.Cloud = append(updated.Client.Whitelist.Cloud, domain)
+	}
```

Comment:
This prevents YAML de-duplication of the global whitelist. Similar to what happens with the list of domain-fronted servers.